backend/entities/Property.cs

```diff
@@ -34,6 +36,7 @@ public abstract class Property : BaseEntity
         /// <summary>
         /// get/set - The status for this property.
         /// </summary>
+        [NotMapped]
         public PropertyStatus Status { get; set; }
```

Comment:
This is awkward.  I generally only do this for transitory data.  I believe this information is only required in models returned from the API.  I see no reason to add it to the entity.

Reply:
Update the relevant API models to include this property.  Don't include it in the entity.